packages/js/components/src/experimental-select-control/select-control.tsx

```diff
@@ -112,8 +120,9 @@ function SelectControl< ItemType = DefaultItemType >( {
 		highlightedIndex,
 		getItemProps,
 		selectItem,
-		selectedItem: singleSelectedItem,
+		selectedItem: selectedItemSingle,
```

Comment:
It confused me a bit what `selectedItemSingle` was referring to versus `singleSelectedItem`, given they are mostly just rearranging the same words :)
Could we change to to `comboboxSingleSelectedItem` or something like that, so we can clearly depict this is from `combobox` internal state?

Reply:
Definitely!  I like this name much better 😄